src/main/java/uk/gov/cslearning/catalogue/config/ElasticSearchConfig.java

```diff
@@ -1,43 +1,37 @@
 package uk.gov.cslearning.catalogue.config;
 
-import org.elasticsearch.client.Client;
-import org.elasticsearch.common.settings.Settings;
-import org.elasticsearch.common.transport.InetSocketTransportAddress;
-import org.elasticsearch.transport.client.PreBuiltTransportClient;
+import com.github.vanroy.springdata.jest.JestElasticsearchTemplate;
+import io.searchbox.client.JestClient;
+import io.searchbox.client.JestClientFactory;
+import io.searchbox.client.config.HttpClientConfig;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.core.io.ClassPathResource;
 import org.springframework.core.io.Resource;
 import org.springframework.data.elasticsearch.core.ElasticsearchOperations;
-import org.springframework.data.elasticsearch.core.ElasticsearchTemplate;
 import org.springframework.data.elasticsearch.repository.config.EnableElasticsearchRepositories;
 import org.springframework.data.repository.init.Jackson2RepositoryPopulatorFactoryBean;
 
-import java.net.InetAddress;
-import java.net.UnknownHostException;
-
 @Configuration
 @EnableElasticsearchRepositories(basePackages = "uk.gov.cslearning.catalogue.repository")
 public class ElasticSearchConfig {
 
     @Bean
-    public Client client(ElasticSearchProperties properties) throws UnknownHostException {
-
-        Settings.Builder builder = Settings.builder();
+    public JestClient jestClient(ElasticSearchProperties properties) {
+        JestClientFactory factory = new JestClientFactory();
 
-        if (properties.getClusterName() != null) {
-            builder.put("cluster.name", properties.getClusterName());
-        }
-        Settings settings = builder.build();
+        String serverUri = "http://" + properties.getHost() + ":" + properties.getPort();
```

Comment:
Probably better to change the config property to 'uri' rather than formatting the uri ourselves.